libs/payments-ui/src/lib/coupon-form/coupon-form.tsx

```diff
@@ -90,16 +92,18 @@ function WithoutCoupon({
       data-testid="coupon-form"
     >
       <div className="input-row">
-        <input
-          className="coupon-input"
-          type="text"
-          name="coupon"
-          data-testid="coupon-input"
-          defaultValue={promotionCode}
-          placeholder="Enter code"
-          disabled={disabled}
-          ref={couponInputRef}
-        />
+        <Localized attrs={{ placeholder: true }} id="coupon-enter-code">
+          <input
+            className="coupon-input"
+            type="text"
+            name="coupon"
+            data-testid="coupon-input"
+            defaultValue={promotionCode}
+            placeholder="Enter code"
+            disabled={disabled}
+            ref={couponInputRef}
```

Comment:
nit: SHould this have a [`maxlength`](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/maxlength)?